phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/LibraryPanel.java

```diff
@@ -58,20 +62,25 @@ private void populateContent() {
         gridPane.getStyleClass().add("grid");
 
         try {
-            JSONObject jsonObject = new JSONObject(shortcut.getScript());
+            System.out.println(shortcut.getScript());
```

Comment:
This looks like a forgotten debug output.
Here we should use `LOGGER` or remove it.